cmd/zar/zq/command.go

```diff
@@ -0,0 +1,238 @@
+package zq
+
+import (
+	"context"
+	"encoding/json"
+	"errors"
+	"flag"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"regexp"
+
+	"github.com/brimsec/zq/archive"
+	"github.com/brimsec/zq/ast"
+	"github.com/brimsec/zq/cmd/zar/root"
+	"github.com/brimsec/zq/driver"
+	"github.com/brimsec/zq/emitter"
+	"github.com/brimsec/zq/pkg/nano"
+	"github.com/brimsec/zq/scanner"
+	"github.com/brimsec/zq/zbuf"
+	"github.com/brimsec/zq/zio"
+	"github.com/brimsec/zq/zio/detector"
+	"github.com/brimsec/zq/zio/ndjsonio"
+	"github.com/brimsec/zq/zng/resolver"
+	"github.com/brimsec/zq/zql"
+	"github.com/mccanne/charm"
+	"go.uber.org/zap"
+)
+
+var Zq = &charm.Spec{
+	Name:  "zq",
+	Usage: "zq [-R dir] [options]",
+	Short: "walk an archive and run zq logic",
```

Comment:
what does "logic" mean here ... could this be more simply expressed as "walk an archive and run zq over it"? (similar comment for the `Long:` section below)

Reply:
I'll tweak it.  It doesn't "run zq".  It runs zql queries similar to how the standalone zql command works.